### ISSUE TITLE
This app

### ISSUE THREAD

**Issue by Naofuminal** [mukul500/AnimeXStream#467]
This app is really shit, it stops working after every single month. Don't use this app just delete it and use any other website with "ads" because whenever someone asks the creator to fix app he says he ain't getting paid.

**Comment by Dragonspartan12** [comment]
you should be grateful you're getting it without ads at all for free in the first place, doesn't matter if it always works or not

**Comment by Naofuminal** [comment]
> you should be grateful you're getting it without ads at all for free in the first place, doesn't matter if it always works or not

Yeah you're right indeed, but the lines I said ain't wrong either

**Comment by Carnaken** [comment]
Use kayuri app, it also doesn't have ads and the creator is so fast the fix the bug of his app.

**Comment by STARGAMERMJ** [comment]
> > Use kayuri app, it also doesn't have ads and the creator is so fast to fix the bug of his app.
> 
> Kayuri is also not working bro

Use the Kayuri application. This app's developer already fixed the problem.

**Comment by Toymakerftw** [comment]
@BlackoftheDawn I get it , u don't like the app. For heaven sake Stop spamming 🙍. The developer has a life too man , unlike some people here. He may or may not fix the app, it upto him. He doesn't owe you anything .

**Comment by BlackoftheDawn** [comment]
> @BlackoftheDawn I get it , u don't like the app. For heaven sake Stop spamming 🙍. The developer has a life too man , unlike some people here. He may or may not fix the app, it upto him. He doesn't owe you anything . 

Wheres the animosity coming from and where do you see spamming. I'm reporting a problem if you can clearly see so that it can get address thus making the app thrive. Seems like the one with no life is the one pointing the finger 😂🤷🏽‍♂️

**Comment by BlackoftheDawn** [comment]
> @BlackoftheDawn Mybad. I was just annoyed with the constant notifications. 

Just change them so that they dont come so frequently or be more selective with what you receive. I did that today to prevent that very thing however if its your job that's something different. I'm merely here to help the community plain and simple. If I was knowledgable at coding I'd offer help however at the moment I've my limitations so I'll leave it to those whom are more skilled and passionate about this to solve the problem. Anyway your good just try to be more understanding in the future that's all I ask.👍🏼

**Comment by Eximinati** [comment]
> 

you all can try zoro.to
